sim/druid/insect_swarm.go

```diff
@@ -0,0 +1,91 @@
+package druid
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+)
+
+const InsectSwarmRanks = 5
+
+var InsectSwarmSpellId = [InsectSwarmRanks + 1]int32{0, 5570, 24974, 24975, 24976, 24977}
+var InsectSwarmsSellDotCoeff = [InsectSwarmRanks + 1]float64{0, .158, .158, .158, .158, .158}
+var InsectSwarmBaseDotDamage = [InsectSwarmRanks + 1]float64{0, 66, 138, 174, 264, 324}
+var InsectSwarmManaCost = [InsectSwarmRanks + 1]float64{0, 45, 85, 100, 140, 160}
+var InsectSwarmLevel = [InsectSwarmRanks + 1]int{0, 1, 30, 40, 50, 60}
+
+func (druid *Druid) registerInsectSwarmSpell() {
+	druid.InsectSwarm = make([]*DruidSpell, InsectSwarmRanks+1)
+
+	for rank := 1; rank <= InsectSwarmRanks; rank++ {
+		config := druid.getInsectSwarmBaseConfig(rank)
+
+		if config.RequiredLevel <= int(druid.Level) {
+			druid.InsectSwarm[rank] = druid.RegisterSpell(Humanoid|Moonkin, config)
+		}
+	}
+}
```

Comment:
🔨 We should probably only register them if the druid has the insect swarm talent selected

Reply:
Sorry I was MIA - shall I close this one?